shared/surface_flux.F90

```diff
@@ -391,7 +413,9 @@ subroutine surface_flux_1d (                                           &
 
   where (avail)
      ! scale momentum drag coefficient on orographic roughness
-     cd_m = cd_m*(log(z_atm/rough_mom+1)/log(z_atm/rough_scale+1))**2
+     where (.not. seawater)
```

Comment:
this seems like it could have implications for many models

Reply:
Actually, this should not affect the results. The cd_m value over seawater points will not be affected by the following line of code because rough_scale is set equal to rough_mom over these points.

cd_m = cd_m*(log(z_atm/rough_mom+1)/log(z_atm/rough_scale+1))**2

Reply:
agreed with Kun. I think I did a test and it does not influence the results. Will further confirm it.

Reply:
What happens elsewhere, i.e., over the ocean? Shouldn't cd_m be defined there?